manifests/a/AntibodySoftware/WizTree/4.01/AntibodySoftware.WizTree.installer.yaml

```diff
@@ -8,6 +8,8 @@ InstallModes:
 - interactive
 - silent
 - silentWithProgress
+InstallerSwitches:
+  InstallLocation: /DIR="<INSTALLPATH>"

```

Comment:
Are these needed?

Since the ```InstallerType``` is ```inno```, it's automatically defined by https://github.com/microsoft/winget-cli/blob/3a4d0071b6d1e3aad1bca0fc8b0f0a59d1c9ba79/src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp#L407.

Reply:
Didn't know about that. 😅